activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -28,6 +31,15 @@ def self.extended(base) # :nodoc:
       end
     end
 
+    # Declares a method that makes it possible to extend components of Rails
```

Comment:
What I was trying to convey with that sentence was that this method allows users to extend different components of rails like `ActionView`, `ActiveRecord` etc. But I may have made it sound like the method is only usuable within Rails apps, I'll try to rephrase the sentence so it makes more sense..